spyder/utils/vcs.py

```diff
@@ -31,7 +32,7 @@
     'name': 'Git',
     'rootdir': '.git',
     'actions': dict(
-        commit=( ('git', ['gui']), ),
+        commit=( ('git', ['gui' if os.name == 'nt' else 'cola']), ),

```

Comment:
What happens if git-cola is not installed?

Reply:
Nothing happens, i.e. no error is generated but also no message is shown. That was the previous behavior.

Would you like to improve it?

Reply:
> Would you like to improve it?

Not at the moment, perhaps at some time in the undetermined future as part of a bigger overhaul.

Reply:
@ccordoba12 As @juanis2112 confirmed, this change appears to silently breaks the `Commit` action in the Files pane context menu on all OSes except Windows unless the third party `git cola` GUI is installed. Is there some reason Spyder can't simply use the built-in `git gui` for this on all platforms, as it did prior to this PR, instead of a third party one that likely only a small minority of users have installed? It should work on any mainstream platform that Git itself is supported on (at least for Git distributions that include it, which is most though not all of them), and I can confirm it works fine on Windows, and appears to work on macOS as well in limited testing, at least with standard `brew`-installed Git. Is there a Spyder-specific reason it cannot be used here on those platforms?